.github/workflows/qe-image.yml

```diff
@@ -0,0 +1,52 @@
+name: Push qe images for new tag
+on:
+  push:
+    tags:
+      - '*'
+env:
+  IMAGE_REGISTRY: quay.io
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+        go:
+          - '1.21'
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v4
+      - name: Log in to Quay.io
+        uses: redhat-actions/podman-login@v1
+        with:
+          username: ${{ secrets.QUAY_IO_USERNAME }}
+          password: ${{ secrets.QUAY_IO_PASSWORD }}
+          registry: ${{ env.IMAGE_REGISTRY }}
+      - name: create script of building and push images
+        run: |
+          cat <<EOF > build-qe-images.sh
```

Comment:
Also can we reduce the number of commands to 4 lines, 2 for e2e (build and push) 2 for integration (build and push) and then use arch and os as vars coming from a matrix (https://github.com/crc-org/crc/blob/main/.github/workflows/windows-artifacts.yml#L47) that will help to reduce the flow

Reply:
updated